server/src/weaverbird/backends/pandas_executor/steps/utils/formula.py

```diff
@@ -15,14 +15,23 @@
 }
 
 
+class NullSeries(Series):
+    """This type allow us to know that a series is filled with None without re-checking all its content."""
```

Comment:
```suggestion
    """A Series containing only null values.
    
    This type allows us to know that a Series is filled with null values without having to check its content
    """
```

https://peps.python.org/pep-0257/